spec/site_prism/page_objects/page/organisations/organisation_courses_as_an_accredited_body.rb

```diff
@@ -0,0 +1,23 @@
+module PageObjects
+  module Page
+    module Organisations
+      class OrganisationCoursesAsAnAccreditedBody < CourseBase
+        set_url "/organisations/{provider_code}/courses_as_an_accredited_body"
```

Comment:
Is this URL correct? If it is then it should be hypenated

Reply:
Good spot, changed to `/organisations/{provider_code}/training_providers/{training_provider_code}/courses`

Reply:
URL's should be dashed/hyphenated - https://www.gov.uk/guidance/content-design/url-standards-for-gov-uk

Reply:
ah sorry, `training-providers` 🤦‍♀